src/scrabble.js

```diff
@@ -1,5 +1,47 @@
-function scrabble() {
+const letterValue = {
+    'A': 1,
+    'B': 3,
+    'C': 3,
+    'D': 2,
+    'E': 1,
+    'F': 4,
+    'G': 2,
+    'H': 4,
+    'I': 1,
+    'J': 8,
+    'K': 5,
+    'L': 1,
+    'M': 3,
+    'N': 1,
+    'O': 1,
+    'P': 3,
+    'Q': 10,
+    'R': 1,
+    'S': 1,
+    'T': 1,
+    'U': 1,
+    'V': 4,
+    'W': 4,
+    'X': 8,
+    'Y': 4,
+    'Z': 10 
+}
 
+function scrabble(word) {
+    let total = 0
+if (word === null){
```

Comment:
This condition seems to be useless 🤔
Null is never returned in the test I did, even calling the function without an argument (the code just breaks).
A condition to avoid code break here would be a good replacement.

Reply:
> This condition seems to be useless 🤔 Null is never returned in the test I did, even calling the function without an argument (the code just breaks). A condition to avoid code break here would be a good replacement.

Naaah, wrong me 😅😅